lib/Fenix/Model/IRC.pm

```diff
@@ -232,6 +240,7 @@ __DATA__
 CREATE TABLE channel (
      channel_id INTEGER PRIMARY KEY AUTOINCREMENT,
      channel TEXT UNIQUE NOT NULL,
+     public INTEGER NOT NULL DEFAULT 0,
```

Comment:
unless you are rebuilding the database from scratch, you have to add this at the end of the migration script ... 

```sql
-- 2 up

ALTER TABLE channel ADD public INTEGER NOT NULL DEFAULT 0;
```